composer.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "openeuropa/oe_contact_forms",
+    "description": "OpenEuropa Drupal module template.",
+    "type": "drupal-module",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "drupal/core": "^8.7",
+        "drupal/contact_storage": "^1.0",
```

Comment:
in OPENEUROPA-3039 its already locked at 1.0.0, lets do it here also to make drone green